lib/releaselib.sh

```diff
@@ -480,23 +480,20 @@ release::gcs::copy_release_artifacts() {
 
   # Copy the main set from staging to destination
   logecho -n "- Copying release artifacts to $gcs_destination: "
```

Comment:
Let's pull in some of the lost messaging
"Copying **public** release artifacts..."